stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/state/querychannels/internal/QueryChannelsMutableState.kt

```diff
@@ -42,6 +44,8 @@ internal class QueryChannelsMutableState(
     latestUsers: StateFlow<Map<String, User>>,
 ) : QueryChannelsState {
 
+    private val logger = ChatLogger.get("QueryChannelsState")
```

Comment:
I wonder if we can use `StreamLog` here since the logging below is for `DEBUG` builds only.
`StreamLog` works in `DEBUG` builds only by default.

```suggestion
    private val logger = StreamLog.getLogger("QueryChannelsState")
```